cognite/client/data_classes/shared.py

```diff
@@ -0,0 +1,25 @@
+from typing import *
+
+from cognite.client.data_classes._base import *
+
+
+# GenClass: EpochTimestampRange
+class EpochTimestampRange(CognitePropertyClass):
+    """Range between two timestamps.
+
+    Args:
+        max (int): The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds.
+        min (int): The number of milliseconds since 00:00:00 Thursday, 1 January 1970, Coordinated Universal Time (UTC), minus leap seconds.
+        cognite_client (CogniteClient): The client to associate with this object.
+    """
+
+    def __init__(self, max: int = None, min: int = None, cognite_client=None):
+        self.max = max
+        self.min = min
+        self._cognite_client = cognite_client
+
+    # GenStop
+    def __getattr__(self, item):
```

Comment:
why not a  @ property or something?

Reply:
The why is me being a python newb. Will fix.

Reply:
actually I think you can leave it out completely and it will work, the defaults on __init__ take care of None, no?

Reply:
Property si having other nice properties I didn't feel like I managed to duplicate with this method as well, so I was quite motivated to fix.